microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -196,10 +196,20 @@ In the Copilot pane, the Compose tool uses AI to write for you, starting from a
 
 To access Compose, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of Microsoft Edge, and then select **Compose**. Text and selections that you enter are sent to an internal Microsoft Large Language Model (LLM). 
 
+
 Requests for Compose are sent over a secure HTTPS connection. Each request contains the input prompt, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.
 
 To change settings for the sidebar or Copilot pane, go to `edge://settings/sidebar`, and then select **App and notification settings** > **Copilot**.
 
+#### Rewrite on any webpage with Compose 
```

Comment:
We should rename the existing Compose section above to reflect that is just about "Compose sidebar".

Reply:
added!